src/cli/cli.cc

```diff
@@ -114,6 +114,10 @@ String getSocketHome (bool verbose) {
   static String LOCALAPPDATA = getEnv("LOCALAPPDATA");
   static String SOCKET_HOME = getEnv("SOCKET_HOME");
   static String HOME = getEnv("HOME");
+  static const bool SSC_DEBUG = (
+    getEnv("SSC_DEBUG").size() > 0 ||
```

Comment:
Nice. I think we also need to look at differentiating between debug output and debug builds in `install.sh` and `install.ps1`.